react-hyf/react-hyf-client/src/components/HomeworkWeek3.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
```

Comment:
You have twice the same file/component in your repo (HomeworkWeek3), could you remove the one which is not being used?